docs/requirements/requirements.md

```diff
@@ -0,0 +1,73 @@
+# Requirements
+
+In this document the terms *'Must have'*, *'Should have'*, *'Could have'* and *'Won't have'* refer to the [MoSCoW method](https://en.wikipedia.org/wiki/MoSCoW_method).
+
+
+### Backend
+| What | How | Importance |Comment|
+|------|-----|------------|-------|
+|Interface|REST-API|Must have| |
+|Platform| Linux | Must have | |
+|Webserver|Tomcat|Should have| |
+|Platform| Windows, OSX | Should have | |
+|Language|Java|Should Have|No node.js (Must have)|
+
+
+### User Interface (UI)
+| What | How | Importance |Comment|
+|------|-----|------------|-------|
+|Ease of Use | Very Simple | Must Have | The UI must be simple and easy to use. Within a handful of clicks the transformation should be achieved |
+|Look and Feel  | The UI must look awesome. No further requirements. | Must Have | |
+|Type|Web Application|Must have| |
+|Supported browsers|Firefox, Chrome| Must have| |
+|Communcation|REST-calls|Must have| |
+|Technology|Angular|Should have| |
+
+### Command Line Interface (CLI)
+| What | How | Importance |Comment|
+|------|-----|------------|-------|
+|Communication|REST-calls|Must have| |
+
+### Transformations
+| What | How | Importance |Comment|
+|------|-----|------------|-------|
+|Target Host OS| Linux | Must have | |
+| Validation of input (CSAR) | syntax | Must Have | | 
+| Validation of input (CSAR) | semantics | Must Have | Not before execution; but if an error is found while executing the transformation, an error must be reported | 
+|TOSCA --> Cloud Foundry| CSAR -> Cloud Foundry Artifacts & Orchestration, if needed|Should have| it is acceptable to optimize the topology |
```

Comment:
> CSAR -> Cloud Foundry Artifacts & Orchestration, if needed

Seems confusing. I assume "if needed" refers to orchestration, but it isn't really apparent. 

> CSAR -> Cloud Foundry Artifacts - And Orchestration, if needed.

Something similar to this would probably be better.